### PR TITLE
New Board based on nRF9160 SiP

### PR DESCRIPTION
**Pull request by olsky**
Dear Team,

I would like to commit the board definition files, the board is arm - nrf9160.

Could you please check?

Thank you & best
Oleh

### CODE REVIEW

**Review by ioannisg — COMMENTED** [review]
Thanks for your contributions @olsky 
To begin, with, could you follow the contribution guidelines, in terms of commit message format?

**Comment by olsky** [comment]
> Thanks for your contributions @olsky
> To begin, with, could you follow the contribution guidelines, in terms of commit message format?

hi @ioannisg sure, thank you for pointing it. i will check and retry!

File: boards/arm/nrf9160_innblue11/doc/index.rst [hunk 1]
```diff
@@ -0,0 +1,203 @@
+.. _nrf9160_innblue11:
+
+nRF9160-INNBLUE11
+################
```
Comment (dbkinder) [inline boards/arm/nrf9160_innblue11/doc/index.rst]: Title underlines need to be at least as long as the title, so add one more ``#``

File: boards/arm/nrf9160_innblue11/doc/index.rst [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`PWM (Pulse Width Modulation)`
+* :abbr:`RTC (nRF RTC System Clock)`
+* Segger RTT (RTT Console)
+* :abbr:`SPI (Serial Peripheral Interface)`
+* :abbr:`UARTE (Universal asynchronous receiver-transmitter with EasyDMA)`
+* :abbr:`WDT (Watchdog Timer)`
+* :abbr:`IDAU (Implementation Defined Attribution Unit)`
+
+.. figure:: img/nrf9160_pca10090.png
```
Comment (dbkinder) [inline boards/arm/nrf9160_innblue11/doc/index.rst]: Change to reference the right image name:  ``img/nrf9160_innblue11.png`` and update the :alt: text and the figure title (``nRF9160 PCA10090 DK (Credit: Nordic Semi)``) to refer to this nrf9160 innblue11 board.

File: boards/arm/nrf9160_innblue11/doc/index.rst [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+     :width: 711px
+     :align: center
+     :alt: nRF9160 PCA10090 DK
+
+     nRF9160 PCA10090 DK (Credit: Nordic Semi)
+
+More information about the board can be found at the
```
Comment (dbkinder) [inline boards/arm/nrf9160_innblue11/doc/index.rst]: I don't see any information about this nRF9160 innblue v11 board at this site or anywhere on on the Nordic site.

File: boards/arm/nrf9160_innblue11/doc/index.rst [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+********
+
+nRF9160 DK has two external oscillators. The frequency of
+the slow clock is 32.768 kHz. The frequency of the main clock
+is 32 MHz.
+
+Supported Features
+==================
+
+The nrf9160_pca10090 board configuration supports the following
```
Comment (dbkinder) [inline boards/arm/nrf9160_innblue11/doc/index.rst]: You've switched board names here. Is this the right information about the nRF9160 innblue v11 board?

File: boards/arm/nrf9160_innblue11/doc/index.rst [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+   $ minicom -D <tty_device> -b 115200
+
+Replace :code:`<tty_device>` with the port where the board nRF9160 DK
+can be found. For example, under Linux, :code:`/dev/ttyACM0`.
+
+Then build and flash the application in the usual way.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: nrf9160_pca10090
```
Comment (dbkinder) [inline boards/arm/nrf9160_innblue11/doc/index.rst]: shouldn't the board be nRF9160_innblue11 ?

**Review by dbkinder — CHANGES_REQUESTED** [review]
documentation looks to be a copy of another board and not fully edited to this nRF9160_innblue11 board?

File: boards/arm/nrf9160_innblue11/Kconfig.defconfig [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+# be restricted to the size of its code partition.
+# For the non-secure version of the board, the firmware
+# must be linked into the code-partition (non-secure) defined in DT, regardless.
+# Apply this configuration below by setting the Kconfig symbols used by
+# the linker according to the information extracted from DT partitions.
+
+if BOARD_NRF9160_INNBLUE11 && TRUSTED_EXECUTION_SECURE
+
+config FLASH_LOAD_SIZE
+	default $(dt_hex_val,DT_CODE_PARTITION_SIZE)
```
Comment (kklimek) [inline boards/arm/nrf9160_innblue11/Kconfig.defconfig]: dt_hex_val function is deprecated, shouldn't it be `default $(dt_chosen_reg_size,$(DT_CHOSEN_Z_CODE_PARTITION))
`

Comment (kklimek) [inline boards/arm/nrf9160_innblue11/Kconfig.defconfig]: `dt_hex_val` function is deprecated, shouldn't it be `default $(dt_chosen_reg_size,$(DT_CHOSEN_Z_CODE_PARTITION))`

File: boards/arm/nrf9160_innblue11/nrf52840_reset.c [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+	if (err) {
+		return err;
+	}
+
+	/* Wait for the nRF52840 peripheral to stop sending data.
+	 *
+	 * It is critical (!) to wait here, so that all bytes
+	 * on the lines are received and drained correctly.
+	 */
+	k_sleep(1);
```
Comment (kklimek) [inline boards/arm/nrf9160_innblue11/nrf52840_reset.c]: use `K_MSEC` macro for sleep interval

**Review by kklimek — CHANGES_REQUESTED** [review]

**Pull request merged**